app/jinja_filters.py

```diff
@@ -190,19 +190,38 @@ def setAttributes(dictionary, attributes):
 
 
 @blueprint.app_template_filter()
-def question_as_legend(question):
+def answers_require_legend(question):
```

Comment:
This seems to be named for an answer argument, but actually operates on a question.